forge-gui/res/cardsfolder/upcoming/sunken_citadel.txt

```diff
@@ -0,0 +1,10 @@
+Name:Sunken Citadel
+ManaCost:no cost
+Types:Land Cave
+K:CARDNAME enters the battlefield tapped.
+K:ETBReplacement:Other:ChooseColor
```

Comment:
this probably should be one replacement effect instead

Reply:
Most land cards like this are done like this. I think this has to do with allowing you too choose the order of replacements effects ?

Reply:
it probably wouldn't make much of a difference when they are done as two replacement effects 🤷‍♂️ 

that could be updated later ... (or when i'm going to remove the `CARDNAME enters the battlefield tapped` keyword)